featuretools/tests/primitive_tests/test_groupby_transform_primitives.py

```diff
@@ -271,7 +271,7 @@ def check(feature):
     for primitive in [CumSum, CumMean, CumMax, CumMin]:
         check(ft.Feature(pd_es, 'log', 'value', groupby=ft.IdentityFeature(pd_es, 'log', 'session_id'), primitive=primitive))
 
-    check(ft.Feature(ft.IdentityFeature(pd_es, 'log', 'session_id'), groupby=ft.IdentityFeature(pd_es, 'log', 'session_id'), primitive=CumCount))
+    check(ft.Feature(ft.Feature(pd_es, 'log', 'product_id'), groupby=ft.Feature(pd_es, 'log', 'product_id'), primitive=CumCount))
```

Comment:
`session_id` is now an `Integer` logical type, so it is not recognized as a valid input for the `CumCount` primitive, so updated to use `product_id` instead, which is `Categorical`.